qa/workunits/mgr/test_prometheus_metrics.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -ex
```

Comment:
There is a lot of piping in this script (to `jq`, `grep`, etc). Lots of things could fail in these pipes. Bash has `set -o pipefail` - do we have to use `/bin/sh` here or an this be a full bash script?